src/theme/components/DisplayCard.ts

```diff
@@ -48,10 +48,10 @@ export const DisplayCard: ComponentMultiStyleConfig = {
       display: "flex",
       justifyContent: "flex-start",
       boxSizing: "border-box",
-      paddingInlineEnd: "2rem",
-      paddingInlineStart: "2rem",
-      paddingTop: "2rem",
-      paddingBottom: "1.5rem",
+      paddingInlineEnd: "1.5rem",
+      paddingInlineStart: "1.5rem",
+      paddingTop: "1rem",
+      paddingBottom: "1rem",
```

Comment:
can i double check on why this was changed?

Reply:
Design wanted the the display cards  from 16 X 16 X 16 X 16 to become 24 X 24 left and right and the top and bottom 16 X 16. 

This is intentionally made as a broad change to allow all other display cards in the dashboard (Ie all display cards since they are not being used anywhere else, correct me if I am wrong about this) to follow this format instead

Reply:
ok if design says is for all display cards then is fine